index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    <div class="container">
+    
+    <header class="portfolio-header">
+    <div class="logo">My Logo</div>
+    
+        <div class="menu-mobile">
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+        <div class="menu-bar"></div>
+    </div>
+
+    </header> 
+  
+    
+
+<h1 class="salutation">
+   Hello. I’m Austin.
+    Glad to see you!</h1>
+<p class="developer">
+    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+</p>
+<h2 class="connect">LET'S CONNECT</h2>
+
```

Comment:
- There are two issues I have mentioned below. Kindly fix those.

- ![image](https://github.com/ausiemass/portfolio-austin/assets/60083437/4d290b7e-40c7-4d82-b205-470df639039f)

- [ ] The style of the contents inside the `h1` tag is not matching with the given design. In the given design it is divided into three lines, but in yours, it is always in two lines. You can consider using `br` to break between the lines.
- [ ] I can see `LET'S CONNECT` is a bit bold in your design. It is not matching with the given design.